core/src/main/scala/gnieh/diffson/package.scala

```diff
@@ -15,6 +15,43 @@
 */
 package gnieh
 
+import scala.collection.immutable.Queue
+
 /** This package contains an implementation of Json JsonPatch, according to [RFC-6902](http://tools.ietf.org/html/rfc6902)
  */
-package object diffson
+package object diffson {
+
+  private val IsDigit = "(0|[1-9][0-9]*)".r
+
+  type Pointer = Queue[Either[String, Int]]
+
+  implicit class PointerOps(val pointer: Pointer) extends AnyVal {
```

Comment:
I can do it on my own, I just not sure if it's really good idea. This way `Pointer` class will only be accessible inside classes extending `JsonSupport` trait. Currently `Pointer` type can be used anywhere outside...